openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -53,7 +53,7 @@ def _create_enrollments(self, *course_run_ids):
 
     def _assert_progress(self, meter, *progresses):
         """Variadic helper used to verify progress calculations."""
-        self.assertEqual(meter.progress, list(progresses))
+        self.assertEqual(meter.progress(), list(progresses))
```

Comment:
i can't figure out how to run this test in paver, the command in the docs is failing for me for some reason